packages/web-app-files/tests/unit/components/SideBar/SideBar.spec.js

```diff
@@ -21,63 +28,151 @@ const simpleOwnFolder = {
   size: '740'
 }
 
+const selectors = {
+  noSelectionInfoPanel: 'noselection-stub'
+}
+
 describe('SideBar', () => {
-  it('fetches file info if 1 item is selected', () => {
-    const mockFileInfo = jest.fn()
-    mockFileInfo.mockReturnValueOnce(Files['/'][1])
-
-    createWrapper({
-      item: simpleOwnFolder,
-      selectedItems: [simpleOwnFolder],
-      mocks: { $client: { files: { fileInfo: mockFileInfo } } }
+  describe('file info', () => {
```

Comment:
wrapped the file info related test cases into its own `describe` block, so the following lines are only indentation changes. Additional tests start in line 96.